rust/main/agents/scraper/src/agent.rs

```diff
@@ -87,7 +87,12 @@ impl BaseAgent for Scraper {
             .settings
             .server(self.core_metrics.clone())
             .expect("Failed to create server");
-        let server_task = server.run().instrument(info_span!("Relayer server"));
+        let server_task = tokio::spawn(
+            async move {
+                server.run();
+            }
+            .instrument(info_span!("Scraper server")),
```

Comment:
Just as FYI: I changed this from "Relayer" to "Scraper" (since we're in the scraper). 

I think that's the correct instrumentation for this code, but lmk if you'd like it reverted.